h/authentication.py

```diff
@@ -21,6 +21,13 @@
 from .oauth import IClientFactory
 
 
+def normalize_authorization_header(request):  # bw compat
+    """Convert any X-Annotator-Auth-Token header to an Authorization header."""
+    token = request.headers.pop('X-Annotator-Auth-Token', None)
```

Comment:
Do we actually want to remove the header here or just read it? I'm just wondering about the value of other pieces of code needing to know if the original header was present. Usually in these cases, a new normalized header is added and used in the code, but the original is left.